components/layout/ContentHeader/ContentHeader.styles.ts

```diff
@@ -0,0 +1,30 @@
+import styled, { css } from "styled-components";
+import { noFlexGapSupport } from "theme/mixins/base";
+import { pxToRem } from "theme/mixins/functions";
+import ContentHeader from "./ContentHeader";
+type Props = React.ComponentProps<typeof ContentHeader>;
+
+export const Wrapper = styled.div<Pick<Props, "headerStyle">>`
```

Comment:
## ContentHeader vs PageHeader
The more I looked at these two components, the more I found a reason to keep both. The problem was that PageHeader was repeating the same styling as ContentHeader - so I've updated ContentHeader to be more versatile. In the designs, this style of header usually includes content on the right hand side. Both the title and the right hand content are underlined.